_data/simple-icons.json

```
@@ -15873,6 +15873,11 @@
             "hex": "000000",
             "source": "https://www.thetileapp.com"
         },
+        {
+            "title": "TIM",
+            "hex": "EB0028",
+            "source": "https://risorse.tim.it/content/dam/flytoco-areapubblica-aemfe/loghi/logo.svg"
```

As per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines, this must be a link to the page containing the SVG - not directly to it.
```suggestion
            "source": "https://tim.it"
```